aderyn_core/src/lib.rs

```diff
@@ -52,21 +52,24 @@ where
 }
 
 fn run_auditor_mode(context: &WorkspaceContext) -> Result<(), Box<dyn Error>> {
-    let auditors_with_instances = get_auditor_detectors()
```

Comment:
@TilakMaddy hopefully this makes it easier to build more auditor-mode detectors with ICF

Reply:
@alexroan I'm yet to look into it ... but on a side note I was wondering if we can bring sarif and auditor mode support to ICF *after* the merge. So basically if you have `--icf -o report.sarif` we will say sarif mode is not supported yet with `--icf`. ...and likewise if you have `--icf --auditor-mode` we will say `auditor-mode` is not supported yet.